app/controllers/api/discussions_controller.rb

```diff
@@ -46,13 +47,15 @@ def set_volume
   private
 
   def visible_records
```

Comment:
This is the original intended usage of visible / public records, I think:

```
def visible_records
  current_user.groups
end

def public_records
  Group.visible_to_public
end
```

whereas in a controller where they want to be the same thing (like, ExploreController, maybe?), we just write `accessible_records`:

```
def accessible_records
  Discussion.visible_to_explore # whether you're logged in or not
end
```

Could pull this out of the gem, but overwriting `accessible_records` is just as good, and I reckon that otherwise we'd just have a lot of `visible_records` methods which look like

```
def visible_records
  current_user.is_logged_in? ? current_user.groups : Group.visible_to_public
end
```
anyway.